.circleci/config.yml

```diff
@@ -19,14 +19,15 @@ jobs:
           paths:
             - .bundle
       - restore_cache:
-          key: libvlc-v2-{{ checksum "compileAndBuildVLCKit.sh" }}
+          key: libvlc-v4-{{ checksum "compileAndBuildVLCKit.sh" }}
```

Comment:
I'm curious why do you need to change that?

Reply:
CircleCI doesn't let you override old caches. With this PR, new directories regarding TVVLCKit must be cached. Also, I went from `v2` to `v4` because I messed up with `v3` :p